sd_mining_core/utils/request_utils.py

```diff
@@ -78,8 +78,9 @@ def submit_job_result(config, miner_id, job, temp_credentials, job_start_time, r
         "inference_latency": inference_latency,
         "upload_latency": upload_latency,
         "identity_address": identity_address,
-        "signature": signature  # Include the signature in the result payload
     }
+    if not config.skip_signature:
```

Comment:
if the skip_signature flag is set, we don't need to generate signature, and we should also skip any `wallet_generator` related operations (I'm worried that the initialization might have some issues if the key file doesn't exist)